nncf/quantization/algorithms/post_training/backend.py

```diff
@@ -0,0 +1,6 @@
+from abc import ABC
+from abc import abstractmethod
+
+
+class PostTrainingBackend(ABC):
```

Comment:
Could you specify `PostTrainingBackend` methods?

Reply:
Update `PostTrainingBackend` methods, while it mostly was inspired by OpenVINO backend, so, probably there is a better way to separate backend methods from the common part.